gonvme_utils.go

```diff
@@ -38,8 +38,12 @@ func (sp *sessionParser) Parse(data []byte) []NVMESession {
 
 		case regexp.MustCompile("^\"Address\"").Match([]byte(line)):
 			targetIP := strings.Split(strings.Fields(line)[2], "=")[1]
-			targetPortal := strings.ReplaceAll(strings.Split(strings.Fields(line)[3], "=")[1], "\"", "")
-			curSession.Portal = targetIP + ":" + targetPortal
+			if strings.Split(strings.Fields(line)[3], "=")[0] == "host_traddr" {
```

Comment:
will this remain the same in all cases, do we have any other better way to compare instead of `"host_traddr"`?

Reply:
The if condition is to differentiate for FC and TCP and incase of FC, it will always be host_traddr which represents the host target address